src/main/java/org/wso2/carbon/inbound/feedep/FeedInject.java

```diff
@@ -60,9 +60,9 @@ public FeedInject(String injectingSeq, String onErrorSeq, boolean sequential,
      * inject the message to the sequence
      */
     public boolean invoke(Object object) {
-        org.apache.synapse.MessageContext messageContext = null;
+        org.apache.synapse.MessageContext msgCtx;
```

Comment:
We would not need to rename messageContext to msgCtx. the former variable name is descriptive.

Reply:
Fixed in [1].

[1] https://github.com/wso2-extensions/esb-inbound-feed/pull/7